frinx-uniconfig/user-guide/uniconfig-operations/immediate-commit-model/readme.md

```diff
@@ -20,6 +20,33 @@ transaction is closed. For reading data, it is necessary to close the
 transaction differently, because no data were changed, so calling a
 commit would be unnecessary.
 
+When calling the 'sync-from-network' RPC, it internally calls
+'replace-config-with-operational'. Note that this only works when 
+using the Immediate Commit Model.
+
+
+## Configuration
+Configurations related to UniConfig transactions is placed in the
```

Comment:
**This is incorrect:** Configurations related to UniConfig transactions is placed in the..

Do Either:
**Configurations** related ... **are** placed in the
or
**Configuration** related ... **is** placed in the